indico/modules/events/registration/templates/management/regform_display.html

```diff
@@ -0,0 +1,173 @@
+                </div>
+                {% trans %}Show all the registrations together, regardless of which form they came from.{% endtrans %}
+            </div>
+            <div class="toolbar right">
+                <label class="switch">
+                    <input id="merge-forms" class="switch-input" type="checkbox" name="merge-forms"
+                        {% if merge_forms %}checked{% endif %}></input>
+                    <span class="switch-label" data-off="No" data-on="Yes"></span>
+                    <span class="switch-handle"></span>
+                </label>
+            </div>
+        </div>
+    </div>
+
+    <div id="participant-list-columns">
+        <p>
+            {% trans -%}
+                Select which registration form will have its entries published in the participant list.
+            {%- endtrans %}
+        </p>
+        <ul id="publish-registrations">
+            {% for form in regforms %}
+            <li>
+                <input id="{{form.id}}" type="checkbox" {% if form.publish_registrations_enabled %}checked{% endif %}></input>
+                {{form.title}}
+            </li>
+            {% endfor %}
+        </ul>
+        <p>
+            {% trans -%}
+                By dragging the column titles, you can choose which fields to
+                display and reorder them.
+            {%- endtrans %}
+            {% trans -%}
+                Custom fields added to registration forms can only be shown if the
+                registration forms are not merged.
+            {%- endtrans %}
+        </p>
+        <div class="i-box titled sortable-column-preview">
+            {{ sortable_lists(_("Shown columns"), enabled_columns,
+                              _("Hidden columns"), disabled_columns) }}
+        </div>
+    </div>
+
+    <div id="registration-forms">
+        <p>
+            {% trans -%}
+                By dragging the form titles, you can choose which to display and reorder them.
+            {%- endtrans %}
+        </p>
+        <div class="i-box titled sortable-form-preview">
+            {{ sortable_lists(_("Shown registration forms"), enabled_forms,
+                              _("Hidden registration forms"), disabled_forms,
+                              form_actions_macro) }}
+        </div>
+    </div>
+
+    <button class="i-button big" type="submit">{% trans %}Save{% endtrans %}</button>
+    <a class="i-button big" href="{{ url_for('.manage_regform_list', event) }}">
+        {% trans %}Back{% endtrans %}
+    </a>
+
+    {{ form_footer(form) }}
+
+    <script>
+        (function () {
+            'use strict';
+            function update() {
+                var mergeForms = $('#merge-forms').prop('checked');
+                $('#participant-list-columns').toggle(mergeForms);
+                $('#registration-forms').toggle(!mergeForms);
+            }
+
+            update();
+            $('#merge-forms').on('change', update);
```

Comment:
Since there are no arguments in the `update()` function, I would do:
```javascript
$('#merge-forms').on('change', update).trigger('change');
```
avoiding line 124.